### ISSUE TITLE
remove separate pybinding CMakeLists.txt (Trac #899)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#893]
Several projects (dataclasses, sim-services, etc) have a separate pybindings CMakeLists.txt file, which should be merged in with the main one.

Migrated from https://code.icecube.wisc.edu/ticket/899
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:13:59", 
    "description": "Several projects (dataclasses, sim-services, etc) have a separate pybindings CMakeLists.txt file, which should be merged in with the main one.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335639558230", 
    "component": "combo core", 
    "summary": "remove separate pybinding CMakeLists.txt", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2015-03-11T22:10:56", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150312T01:22:11: nega commented:

> 
> ```text
> ~/i3/combo/build 7s
> ❯ find ../src/ -name CMakeLists.txt  |grep pybindings
> ../src/cmc/private/pybindings/CMakeLists.txt
> ../src/dst/private/pybindings/CMakeLists.txt
> ../src/tpx/private/pybindings/CMakeLists.txt
> ../src/I3Db/private/pybindings/CMakeLists.txt
> ../src/IceHive/private/pybindings/CMakeLists.txt
> ../src/clsim/private/pybindings/CMakeLists.txt
> ../src/ddddr/private/pybindings/CMakeLists.txt
> ../src/CascadeVariables/private/pybindings/CMakeLists.txt
> ../src/diplopia/private/pybindings/CMakeLists.txt
> ../src/CommonVariables/private/pybindings/track_characteristics/CMakeLists.txt
> ../src/CommonVariables/private/pybindings/hit_multiplicity/CMakeLists.txt
> ../src/CommonVariables/private/pybindings/CMakeLists.txt
> ../src/CommonVariables/private/pybindings/hit_statistics/CMakeLists.txt
> ../src/CommonVariables/private/pybindings/direct_hits/CMakeLists.txt
> ../src/neutrino-generator/private/pybindings/CMakeLists.txt
> ../src/DOMLauncher/private/pybindings/CMakeLists.txt
> ../src/wimpsim-reader/private/pybindings/CMakeLists.txt
> ../src/STTools/private/pybindings/CMakeLists.txt
> ../src/finiteReco/private/pybindings/CMakeLists.txt
> ../src/wavereform/private/pybindings/CMakeLists.txt
> ../src/CoincSuite/private/pybindings/CMakeLists.txt
> ../src/cramer-rao/private/pybindings/CMakeLists.txt
> ../src/NoiseEngine/private/pybindings/CMakeLists.txt
> ../src/dataclasses/private/pybindings/CMakeLists.txt
> ../src/topsimulator/private/pybindings/CMakeLists.txt
> ../src/dipolefit/private/pybindings/CMakeLists.txt
> ../src/tensor-of-inertia/private/pybindings/CMakeLists.txt
> ../src/portia/private/pybindings/CMakeLists.txt
> ../src/shield/private/pybindings/CMakeLists.txt
> ../src/toprec/private/pybindings/CMakeLists.txt
> ../src/icetray/private/test-pybindings/CMakeLists.txt
> ../src/icetray/private/pybindings/CMakeLists.txt
> ../src/ophelia/private/pybindings/CMakeLists.txt
> ../src/genie-icetray/private/pybindings/CMakeLists.txt
> ../src/sim-services/private/pybindings/CMakeLists.txt
> ../src/earthmodel-service/private/pybindings/CMakeLists.txt
> ../src/vuvuzela/private/pybindings/CMakeLists.txt
> ```
>

**Comment by icecube-trac** [comment]
Trac update at 20150810T22:58:44: olivas changed milestone from "" to "offline V15-08-00"

**Comment by icecube-trac** [comment]
Trac update at 20150810T23:56:23: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150820T16:59:53: 
* david.schultz changed owner from "olivas" to "david.schultz"
* david.schultz commented:

> steal from olivas so we can get a release faster. also, test nega's ticket folding on #software.
> 
> completed with r2233/IceTray:
> 
> ```text
> ../src/icetray/private/test-pybindings/CMakeLists.txt
> ../src/icetray/private/pybindings/CMakeLists.txt
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150820T19:48:20: 
* david.schultz commented:

> In changeset 136661:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136661"
> consolidate CMakeLists.txt. fixes #893
> ```


* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas commented:

> Milestone offline V15-08-00 deleted


* olivas changed milestone from "offline V15-08-00" to ""
